resources/user-agents/browsers/chedot/chedot-43-0.json

```diff
@@ -47,7 +49,9 @@
             "Win10_x64", "Win10_64", "Win10_32",
             "Win8_1_x64", "Win8_1_64", "Win8_1_32",
             "Win8_x64", "Win8_64", "Win8_32",
-            "Win7_x64", "Win7_64", "Win7_32"
+            "Win7_x64", "Win7_64", "Win7_32",
+            "Vista_64", "Vista_32",
```

Comment:
Ah, and I did add Win Vista and XP to these patterns because their site mentions that the browser is available for these platforms: http://landing.chedot.com